app/Views/ScBooking/Index.cshtml

```diff
@@ -93,7 +93,6 @@
                 @Html.HiddenFor(m => m.CaseSearchResults[i].courtFileNumber)
                 @Html.HiddenFor(m => m.CaseSearchResults[i].styleOfCause)
                 @Html.HiddenFor(m => m.CaseSearchResults[i].physicalFileId)
-                @Html.HiddenFor(m => m.CaseSearchResults[i].courtLevelCode)
```

Comment:
This wasn't being used for anything, so I removed it.  There are 8 properties on the object and only 4 are needed in the hidden fields for handling form error states.